### PR TITLE
Updated objectives for SQL lessons

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: unit2-java/lesson13.rst [hunk 1]
```diff
@@ -7,11 +7,11 @@ Goals
...
+- Explain what a table is.
 - Perform CRUD (Create, Read, Update, Delete) operations using SQL queries
 - Explain what joins are used for
 - Describe the difference between the four types of joins: inner, left outer, right outer, and full outer
-- Write queries using joins
-- Utilize foreign keys
-- Explain the similarities between Python objects and database tables
-- Create tables with an understanding of one-to-many relationships
+- Write simple queries using joins
+- Import tables using a `.sql` file and *MySQL Workbench*
```
Comment (jimflores5) [inline unit2-java/lesson13.rst]: Technically, the students did not import a table in this chapter. Instead they created the tables they needed and then downloaded a ``.sql`` file that filled those tables.

I'd shift this objective to the next lesson.

File: unit2-java/lesson13.rst [hunk 2]
```diff
@@ -7,11 +7,11 @@ Goals
...
+- Explain what relational databases are used for and what SQL is.
+- Explain what a table is.
 - Perform CRUD (Create, Read, Update, Delete) operations using SQL queries
 - Explain what joins are used for
 - Describe the difference between the four types of joins: inner, left outer, right outer, and full outer
-- Write queries using joins
-- Utilize foreign keys
-- Explain the similarities between Python objects and database tables
-- Create tables with an understanding of one-to-many relationships
+- Write simple queries using joins
```
Comment (jimflores5) [inline unit2-java/lesson13.rst]: This lesson only has the students practice inner joins.  We will probably complete the rest of this objective in the next lesson.

File: unit2-java/lesson14.rst [hunk 3]
```diff
@@ -7,8 +7,9 @@ Goals
...
 ----------
 
-- Utilize *phpMyAdmin* and build a database locally
 - Create a new user and database
-- Import tables using a `.sql` file
 - Explain how and why data is split into related tables
+- Utilize foreign keys
+- Create tables with an understanding of one-to-many relationships
 - Perform complex SQL queries with joins and subqueries
+
```
Comment (jimflores5) [inline unit2-java/lesson14.rst:20]: How about two additional objectives to round out the SQL lessons?

1. Import tables using a `.sql` file and *MySQL Workbench*
2. Write simple queries using LEFT, RIGHT, and FULL joins.

**Review by jimflores5 — APPROVED** [review]

File: unit2-java/lesson13.rst (hunk 1 above)
Comment (gildedgardenia) [reply unit2-java/lesson13.rst]: Done!

File: unit2-java/lesson14.rst [hunk 4]
```diff
@@ -4,11 +4,17 @@ Unit 2 (Java) - Lesson 14 Learning Objectives
...
 
-- Utilize *phpMyAdmin* and build a database locally
-- Create a new user and database
-- Import tables using a `.sql` file
 - Explain how and why data is split into related tables
+- Utilize foreign keys
+- Create tables with an understanding of one-to-many relationships
 - Perform complex SQL queries with joins and subqueries
+- Import table data with *MySQL Workbench*
+- Write simple queries using left, right, and full joins
```
Comment (jimflores5) [inline unit2-java/lesson14.rst]: Remove "simple", since joins fall under the "complex" category for queries.

**Review by jimflores5 — APPROVED** [review]
Besides the lesson numbers and one word removal, these are good to merge.

File: unit2-java/lesson14.rst (hunk 4 above)
Comment (gildedgardenia) [reply unit2-java/lesson14.rst]: Done!

**Pull request merged**